Arduino/libraries/Atmega_servo_response/Atmega_servo_response.cpp

```diff
@@ -0,0 +1,112 @@
+#include "Atmega_servo_response.h"
+
+Servo servo1;
+Servo servo2;
+SimpleTimer timer;
+millisDelay srvoDelay;
+
+char doorUnlocked[15] = "Door Unlocked";
+char doorLocked[15] = "Door Locked";
+char rxChar= 0;         // RXcHAR holds the received command.
+int doorState; // 1 for locked, 0  for unlocked
+
+void Wifisetup() {
+  Serial.begin(115200);
+  Serial3.begin(115200);
+  pinMode(LED, OUTPUT); // Sets pin 13 as OUTPUT.
+}
+
+void Wifiloop(){
+  timer.run();
+  //if (srvoDelay.justFinished())           //Check if AutoLock Timer Finished
```

Comment:
void loop() has been transformed to void Wifiloop() as is the standard for our other modules. Keep in mind that this is a function thus in needs to be placed in a while-loop to remain active.